### PR TITLE
Replace modified files on Samsung/Tizen.NET with files on jboothman/Tizen.NET

### PR DESCRIPTION
**Pull request by jboothman**
Replace 22 files on Samsung/Tizen.NET with files on jboothman/Tizen.NET. The files were uploaded and committed to jboothman/Tizen.NET on May 29, 2019.

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (JoonghyunCho) [inline FAQ.md]: The file path for FAQ.md file is moved.  Please move back under _faq folder.

File: _issues/tv-tools-vs.md [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
+
+## Issue on Visual Studio 2019
+After updating Visual Studio 2019 to version 16.1.0, the following build error for your Tizen project appears:
+
+> The TargetFramework value 'tizen40' was not recognized. It may be misspelled. If not, then the TargetFrameworkIdentifier and/or TargetFrameworkVersion properties must be specified explicitly.
+
+### This issue will be fixed soon
+- [This issue](https://github.com/dotnet/project-system/issues/4854) has been [fixed](https://github.com/dotnet/project-system/pull/4859).
+- A fixed version of Visual Studio will be released soon.
+- The Tizen.NET SDK will be fixed and released soon.
```
Comment (JoonghyunCho) [inline _issues/tv-tools-vs.md]: Tizen.NET.Sdk is the package which will be fixed.
We may need to keep the package name. Or change it to like.. "Tizen.NET.Sdk package will be fixed and released soon," ??

File: _issues/wearable-tools-tizenstudio.md [hunk 3]
```diff
@@ -1,34 +1,31 @@
...
+toc_label: Known Issues and Workarounds
+---
+
+
+## Connection issue between a watch and Device Manager
+
+Refer to [Testing Your App on Galaxy Watch][testing_app_on_watch] for information about developing your application and testing it on your watch.
+
+Even if you follow every step correctly, you may face the following error:
+
```
Comment (JoonghyunCho) [inline _issues/wearable-tools-tizenstudio.md]: Please keep the image or leave the error message same.
The following error message and explanation is the popup content what user will see, and we cannot change the content.

File: _issues/tv-tools-vs.md (hunk 2 above)
Comment (jboothman) [reply _issues/tv-tools-vs.md]: I like your rewording. Let's use "The Tizen.NET.Sdk package will be fixed and released soon." Sound OK?

File: _issues/wearable-tools-tizenstudio.md (hunk 3 above)
Comment (jboothman) [reply _issues/wearable-tools-tizenstudio.md]: My apologies. I've reverted my changes to include the illustration and unchanged text. Added modified _issues/wearable-tools-tizenstudio.md file to my branch.

File: FAQ.md (hunk 1 above)
Comment (jboothman) [reply FAQ.md]: Done.

File: _issues/tv-tools-vs.md (hunk 2 above)
Comment (JoonghyunCho) [reply _issues/tv-tools-vs.md]: sound great.

**Pull request merged**